src/Tools/dotnet-user-secrets/test/SecretManagerTests.cs

```diff
@@ -164,7 +164,6 @@ public void SetSecret_Update_Existing_Secret()
         }
 
         [Fact]
-        [QuarantinedTest]
```

Comment:
Failure when quarantined was `malloc_consolidate(): invalid chunk size`, that would be an issue in Runtime, but no other info was captured.

Propose un-quarantining, and if it fails again seeing if we captured a dump or figuring out if we need to add dump capturing.